pkg/dashboard/templates/cost_settings.gohtml

```diff
@@ -0,0 +1,51 @@
+{{ define "cost_settings" }}
+<aside class="cost-settings-box" id="cost-settings-box">
+  <details open>
+    <summary class="--hideMarker">
+      <div class="cost-settings__title-container">
+        <p class="cost-settings-box__toggle-title">
+          Cost Settings
+          <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+          ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+        </p>
+        <div>
+         <button id="cost-settings__disable-cost-settings">Disable Cost Settings</button>
+        </div>
+      </div>
+    </summary>
+
+    <form class="cost-settings-box__content">
+      <p>Choose an AWS or GCP instance type to get an accurate cost data, or set your cost per CPU and cost per GB of memory manually. Pick "Other" Cloud Provider to customize the numbers.</p>
+      <div class="cost-settings-box__options-container">
+        <div class="cost-settings-box__cloud-providers">
+          <label for="cloud-providers">Cloud Provider</label>
+          <div class="cost-settings-box__select-container">
+            <select name="cloud-providers" id="cost-settings-box__cloud-providers">
+            </select>
+            <img src="/static/images/triangle.svg" alt="Select Icon" />
+          </div>
+        </div>
+        <div class="cost-settings-box__instance-types">
+          <label for="instance-types">Instance Type</label>
+          <div class="cost-settings-box__select-container">
+            <select name="instance-types" id="cost-settings-box__instance-types">
+            </select> 
+            <img src="/static/images/triangle.svg" alt="Select Icon" />
```

Comment:
Since this is purely a visual enhancement, and isn't needed for usability, should it be hidden from screen readers?

Reply:
(this applies to the other box as well)